### DISCUSSION TITLE
Select multiple with no selection returns first option

### DISCUSSION THREAD

**Discussion by mwx23** [lona-web-org/lona#165] [Q&A] [answered]
I've noticed that when using a `Select(multiple=True)` if you deselect all options and get it's `.value` it defaults to returning the first option.

The behaviour comes from: 

https://github.com/lona-web-org/lona/blob/d93460a744b6a1057f38a1e66df866899b9e7ba9/lona/html/data_binding/select.py#L128-L131

My expectation is if I have deselected all options then `.value` would be empty list.

I'm happy to override this with a subclass just wondering what the rationale was.

**Comment by fscherf** [comment]
Hi @mwx23!
That's a Bug! A select has always something selected. If nothing is selected by the user, the first option gets used. That does not apply to multi selects though. In this case, if nothing is selected, ``Select.value`` should return an empty list.

I created #166 and started working on a pull request.

Thanks for reporting! :)

**Comment by fscherf** [comment, marked as answer]
@mwx23 This issue is fixed in 1.7.5

**Reply by mwx23** [reply]
@fscherf that's great! I actually noticed the new release earlier today and updated and removed my little shim. Our code was very similar but yours was much better :)

I'm not sure whether you'd prefer things like that in the discussions or in issues. At the moment I'm leaning towards discussions because I'm not sure if some behaviour is intended or not.

In the coming weeks I hope to have a few widgets to share in the Show and Tell section.

Keep up the awesome work!

**Reply by fscherf** [reply]
@mwx23 Great! :)

Discussions are still an experiment and i am not sure either :D Discussions are great for issues that require threads, but one down side is that you can't really close an discussion. I think discussions are better suited for feature requests than issues, because the lead to long threads in the past. Issues are better suited for bug reports, because you can close them when the bug is fixed.
Open what you think is best suited for your issue, in doubt i can transfer an issue to an discussion and vice versa :)

Ouh! :) I am looking forward to it :)
